Integrations/DTO/CreateBrokerData.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DTO
+{
+    /// <summary>
+    /// Required for create any bank broker
+    /// </summary>
+    public class CreateBrokerData
+    {
+        /// <summary>
+        /// Token
+        /// </summary>
+        public string Token { get; set; }
```

Comment:
3 lines of comment to write only one word that copies the name of property - it is strange. 
By the way, I don't get which token it is. Token of a broker? What does it mean - it can give us access to trade?

Reply:
It's called a tag and that's a good way to comment class properties, methods, fields etc., so it's okay i guess.